tasks/main.yml

```diff
@@ -1,27 +1,90 @@
 ---
 # tasks file for ansible-role-gridinit
+
+- include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+  tags: install
+
+- name: Install repository
+  include_tasks: "openio_install_repo_{{ ansible_os_family }}.yml"
+  when: openio_gridinit_repo_managed
+  tags: install
+
 - name: Install packages
   package:
     name: "{{ item }}"
     state: present
-  with_items: "{{ gridinit_packages }}"
+  with_items: "{{ openio_gridinit_packages }}"
+  tags: install
 
-- name: Configuration
-  template:
-    src: gridinit.conf.j2
-    dest: "{{ gridinit_config_file }}"
+
+- name: Create the group openio
+  group:
+    name: openio
+    gid: "{{ openio_group_openio_gid | default('220')}}"
```

Comment:
Why not use defaults/main.yml for default values ?

Reply:
This is not an option, 120/220 is the uid/gid openio.
But in a global playbook, where a variable `openio_group_openio_gid` is defined: all roles inherit the value